src/InlineMethod/InlineBehavior.cs

```diff
@@ -0,0 +1,21 @@
+﻿namespace InlineMethod
+{
+    /// <summary>
+    /// InlineMethod behavior
+    /// </summary>
+    public enum InlineBehavior
+    {
+        /// <summary>
+        /// Keep method after inline
+        /// </summary>
+        Keep,
```

Comment:
I think it's worth assigning numbers to enum values since we're using them above